perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/client/resources/PerunRegistrarTranslation.java

```diff
@@ -333,7 +333,13 @@
 	@DefaultMessage("<br/>You don`t have required Level of Assurance (LoA) to display registration form. Contact your IDP ({0}) and prove your identity to him by official ID card (passport, driving license). If you have another identity with higher Level of Assurance, try to use it instead.")
 	public String cantBecomeMemberInsufficientLoa(String idpName);
 
-	@DefaultMessage("<h4>You can`t submit the registration</h4><p><br/>The Identity Provider you used to log-in ({0}) doesn`t provide data required by the registration form. Please ask your IDP to publish following attributes or use different IDP.")
+	@DefaultMessage("<h4>You can`t submit the registration</h4><p><br/>The following required items have no prefilled value, but they cannot be edited. Please contact the administrator.")
+	public String noPrefilledUneditableRequiredData();
+
+	@DefaultMessage("Required empty item: {0}")
+	public String requiredEmptyItem(String attributeName);
+
+	@DefaultMessage("<h4>You can`t submit the registration</h4><p><br/>The administrator set up this form wrongly OR you don`t match any conditions OR the Identity Provider you used to log-in ({0}) doesn`t provide data required by the registration form. Please contact the administrator eventually ask your IDP to publish the following attributes or use different IDP.")
```

Comment:
I probably wouldn't understand the option `you don`t match any conditions`. Also I would order it from the most probable which is case of IdP not providing the data.